changelogs/fragments/ldap-inventory.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- Added ``microsoft.ad.ldap`` inventory plugin to retreive hosts from an Active Directory source
```

Comment:
I completely forgot about his, has been too long since I've added a new plugin. I'll probably remove it in favour of setting this under the `release_summary` when it comes time to doing the release.